go-client/README.md

```diff
@@ -0,0 +1,47 @@
+# Deephaven Go Client
+
+This is a Go package that provides a client interface for [Deephaven Community Core](https://github.com/deephaven/deephaven-core).
+Details on usage can be found in the examples and in the documentation (both listed below).
```

Comment:
We should have a link to the godoc published on our web once that action is created.